conf/environments.js

```diff
@@ -26,6 +26,16 @@ module.exports = {
             globalReturn: true
         }
     },
+    commonjs: {
+        globals: {
+            module: false,
+            require: false,
+            exports: false
+        },
+        ecmaFeatures: {
+            globalReturn: true
```

Comment:
Ist the top-level function scope really defined in the commonjs spec? I took a brief look at and couldn't find that part.